iothub/device/src/Authentication/IotHubConnectionCredentials.cs

```diff
@@ -310,25 +310,6 @@ private void Validate()
                 throw new FormatException("Missing IoT hub host name.");
             }
 
-            // Host name
-            if (string.IsNullOrWhiteSpace(HostName))
```

Comment:
These checks are already performed in AuthenticationMethod.Populate(), SetAuthenticationModel(), GetAuthenticationMethodFromConnectionString(), PopulatePropertiesFromConnectionString(), SetTokenRefresherIfApplicable().

Thus these 'IsNullorWhiteSpace' checks are never performed in this Validate() method